### ISSUE TITLE
io.reactivex.exceptions.OnErrorNotImplementedException: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | java.lang.NullPointerException

### ISSUE THREAD

**Issue by kutear** [skylot/jadx#1676]
Please describe what you did before the error occurred.
**IMPORTANT!** If the error occurs with a specific APK file please attach or provide link to apk file!
 * Jadx version: 1.4.1
* Java version: 1.8.0_292
* Java VM: AdoptOpenJDK OpenJDK 64-Bit Server VM
* Platform: Mac OS X (10.16 x86_64)
* Max heap size: 22940 MB
* Program args: -Xms128M -XX:MaxRAMPercentage=70.0 -XX:+UseG1GC -Dawt.useSystemAAFontSettings=lcd -Dswing.aatext=true

```
io.reactivex.exceptions.OnErrorNotImplementedException: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | java.lang.NullPointerException
	at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:704)
	at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:701)
	at io.reactivex.internal.subscribers.LambdaSubscriber.onError(LambdaSubscriber.java:79)
	at io.reactivex.internal.subscribers.LambdaSubscriber.onNext(LambdaSubscriber.java:69)
	at io.reactivex.internal.operators.flowable.FlowableObserveOn$ObserveOnSubscriber.runAsync(FlowableObserveOn.java:407)
	at io.reactivex.internal.operators.flowable.FlowableObserveOn$BaseObserveOnSubscriber.run(FlowableObserveOn.java:176)
	at hu.akarnokd.rxjava2.swing.AsyncSwingScheduler$AsyncSwingWorker$WorkerTask.run(AsyncSwingScheduler.java:163)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.NullPointerException
	at java.util.concurrent.ConcurrentHashMap.compute(ConcurrentHashMap.java:1837)
	at jadx.api.JadxDecompiler.convertClassNode(JadxDecompiler.java:476)
	at jadx.api.JadxDecompiler.lambda$convertClassNode$7(JadxDecompiler.java:482)
	at java.util.concurrent.ConcurrentHashMap.compute(ConcurrentHashMap.java:1853)
	at jadx.api.JadxDecompiler.convertClassNode(JadxDecompiler.java:476)
	at jadx.core.utils.Utils.collectionMap(Utils.java:207)
	at jadx.api.JadxDecompiler.getClassesWithInners(JadxDecompiler.java:392)
	at jadx.gui.JadxWrapper.getIncludedClassesWithInners(JadxWrapper.java:126)
	at jadx.gui.ui.dialog.SearchDialog.buildSearch(SearchDialog.java:359)
	at jadx.gui.ui.dialog.SearchDialog.search(SearchDialog.java:334)
	at io.reactivex.internal.subscribers.LambdaSubscriber.onNext(LambdaSubscriber.java:65)
	... 17 more

```

**Comment by jpstotz** [comment]
@kutear Can you please provide more details on what you did when the exception occurred? I can see that you were performing a search on all classes but that is all I know. Does this problem always occur? If yes please check if it still occurs if you upgrade to the latest version of Jadx. 

If it only happens when a specific APK file is loaded, would it be possible to provide that APK file (w.g. rename to zip and attach it to this issue)?

**Comment by skylot** [comment]
Looks like this issue was resolved by change made for #1775 i.e. 4d00fed, so it can be closed.
